compensations/pom.xml

```diff
@@ -300,7 +312,7 @@
                             <redirectTestOutputToFile>true</redirectTestOutputToFile>
                             <skip>false</skip>
                             <systemPropertyVariables combine.children="append">
-                                <!--
```

Comment:
@mayankkunwar please can you revert this formatting?

Reply:
Okay